docs/.vuepress/config.js

```diff
@@ -132,5 +167,10 @@ module.exports = {
   plugins: [
     '@vuepress/plugin-back-to-top',
     '@vuepress/plugin-medium-zoom',
-  ]
+  ],
+  markdown: {
+    extendMarkdown: md => {
+      md.use(require('./markdown-it-gh'))
+    }
+  }
```

Comment:
This allows code snippets using the same format with cosmos sdk. Slightly modified from the cosmos vuepress theme: https://github.com/cosmos/vuepress-theme-cosmos/blob/develop/markdown-it-gh.js